### PR TITLE
Move content out of frontmatter

### PR DESCRIPTION
**Pull request by pdehaan**

### CODE REVIEW

File: .eleventy.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module.exports = (eleventyConfig) => {
+  eleventyConfig.addFilter("keys", o => Object.keys(o));
+
+  return {
+    dir: {
+      input: "src",
+      output: "www",
+    }
+  };
+};
```
Comment (pdehaan) [inline .eleventy.js:10]: Just a basic Eleventy config. THe custom filter was just for debugging, you can ✂️ it.

File: .gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
 node_modules
+www
```
Comment (pdehaan) [inline .gitignore:2]: This is the directory that the generated Eleventy site will go into. Defined in the .eleventy.js#L7 file above.
This just prevents the generated site from getting committed into the repo.

File: package.json [hunk 3]
```diff
@@ -4,6 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
+    "build": "eleventy",
```
Comment (pdehaan) [inline package.json:7]: Easier for me to build the site if I can just do <kbd>npm run build</kbd> from the root directory here (plus, it shows up as an "npm script" in my VS Code).

File: src/_includes/project.html [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <article>
     <h2>{{ project.data.title }}</h2>
-        {{ project.data.content }}
+        {{ project.templateContent }}
```
Comment (pdehaan) [inline src/_includes/project.html:3]: Use the processed Markdown `templateContent` vs the `data.content` frontmatter.

File: src/projects/data/nightly_strategy.md [hunk 5]
```diff
@@ -4,5 +4,6 @@ tags: data
 link_href: https://drive.google.com/file/d/1OdYMe0eRVSR79DCtTiGXHCAcaz91Jdyj/view?usp=sharing
 source_href: https://github.com/tombohub/stock-charts/blob/master/src/scripts/nightly_backtest.py
 tech: Python, Pandas, Pinkfish
-content: Backtesting of 5000+ stocks for overnight strategy. Creating plots comparing it to benchmarks
 ---
+
+Backtesting of 5000+ stocks for overnight strategy. Creating plots comparing it to benchmarks
```
Comment (pdehaan) [inline src/projects/data/nightly_strategy.md:9]: Rest of these diffs are just converting the frontmatter `content` props into page `templateContent` to make rich formatting easier.

**Review by tombohub — APPROVED** [review]

**Comment by tombohub** [comment]
There are some kind of conflict, I am not gonna deal with that now so I am just gonna close it.
Thanks!

**Pull request closed without merging**